src/side-navigation/side-navigation.scss

```diff
@@ -1,207 +1,132 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
-@import '../icons/index';
+@import '../icon/index';
 
 @mixin iui-side-navigation {
   display: inline-flex;
   flex: 0 0 auto;
   flex-direction: column;
-  overflow: hidden;
   transition: max-width $iui-speed-fast ease-out;
   height: 100%;
-
   @include themed {
-    background-color: t(iui-color-background-1);
-    border: 1px solid t(iui-color-background-5);
+    background-color: t(iui-color-background-3);
+    border-right: 1px solid t(iui-color-background-5);
   }
 
-  > .iui-expand-container {
-    height: $iui-l;
-    cursor: pointer;
-    display: inline-flex;
-    justify-content: center;
-    align-items: center;
-
-    @include themed {
-      background-color: t(iui-color-background-1);
-    }
-
-    > .iui-icon {
-      width: $iui-icons-small;
-      height: $iui-icons-small;
-
-      @include themed {
-        fill: t(iui-icons-color);
-      }
-    }
-
-    &:hover > .iui-icon {
-      @include themed {
-        fill: t(iui-icons-color-primary);
-      }
-    }
-
-    &:focus {
-      outline: none;
-    }
-
-    &:focus-visible {
-      @include themed {
-        box-shadow: inset 0 0 $iui-xs
-          rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-3));
-      }
-    }
-  }
-
-  > .iui-sidenav-content {
+  > .iui-content {
     display: flex;
-    flex-direction: row;
-    height: calc(100% - #{$iui-l});
+    flex-direction: column;
+    flex-grow: 1;
+    width: 100%;
 
-    > .iui-sidenav-items-container {
+    > .iui-top {
+      flex-grow: 1;
       display: flex;
       flex-direction: column;
 
-      > .iui-sidenav-items {
-        flex-grow: 1;
-        display: flex;
-        flex-direction: column;
-      }
-
-      > .iui-sidenav-items > .iui-sidenav-item-container,
-      > .iui-bottom-container > .iui-sidenav-item-container {
-        margin-bottom: -1px;
-        @include iui-sidenav-item-container;
+      > .iui-button:not(:last-of-type) {
+        border-bottom: none;
       }
     }
+    
+    > .iui-bottom > .iui-button {
+      border-bottom: none;
+    }
   }
 
   &.iui-collapsed {
-    min-width: $iui-l * 2;
-    max-width: $iui-l * 2;
+    min-width: $iui-icons-large * 3;
+    max-width: $iui-icons-large * 3;
   }
 
   &.iui-expanded,
```

Comment:
Maybe worth having these under the same `&.iui-expanded`?
![image](https://user-images.githubusercontent.com/36186912/118229073-02cbc400-b494-11eb-8be4-40377623c18f.png)
![image](https://user-images.githubusercontent.com/36186912/118229085-0a8b6880-b494-11eb-8ede-59f1eefecfde.png)

Reply:
`.iui-expand` is now within `&.iui-expanded,  > .iui-content {`.